src/SUMMARY.md

```diff
@@ -0,0 +1,16 @@
+# Summary
+
+[LazyLedger Specifications](./README.md)
+
+- [Specification](./specs/README.md)
```

Comment:
Same, can we add some content here?
![image](https://user-images.githubusercontent.com/7441766/120081708-3a0ab980-c0bf-11eb-88c9-365456e34297.png)

Reply:
Fixed in a3be997.